src/components/OurCourses/useStyles.js

```diff
@@ -0,0 +1,51 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(({ typography, breakpoints }) => ({
+  root: {
+    backgroundColor: "#DFDFDF",
+  },
+  section: {
+    paddingTop: typography.pxToRem(60),
+    paddingLeft: typography.pxToRem(20),
+    paddingRight: typography.pxToRem(20),
+    [breakpoints.up("md")]: {
+      padding: `${typography.pxToRem(80)} 0`,
+    },
+  },
+  title: {
+    fontWeight: "900",
+    fontSize: typography.pxToRem(20),
+    marginBottom: typography.pxToRem(40),
```

Comment:
This is already defined in [theme](https://github.com/CodeForAfrica/PesaYetu/blob/08f271db28c6417e188bfb0c0ebd7375684465e7/src/theme/index.js#L141-L145) @gertie-sheshe

Reply:
Fixed